fastlane/Fastfile

```diff
@@ -428,4 +428,44 @@ platform :ios do
 
   end
 
+  desc "Generate release podspecs"
+  lane :generate_release_podspecs do |options|
```

Comment:
[Request] Calling it `publish_podspec` is in line with the name of the existing `publish_swift_package` lane.
```suggestion
  lane :publish_podspec do |options|
```